src/lib/replication/log_store/home_raft_log_store.h

```diff
@@ -199,6 +199,10 @@ class HomeRaftLogStore : public nuraft::log_store {
     nuraft::ptr< nuraft::log_entry > m_dummy_log_entry;
     store_lsn_t m_last_durable_lsn{-1};
     folly::Future< folly::Unit > m_log_store_future;
+
+    // raft log entry cache related members
+    std::shared_mutex m_mutex;
```

Comment:
can we use standard mutex , or we need folly mutex for fiber?

Reply:
this is a mutex in c++ std library , which can be used as a RW lock. we can not use fiber mutex, which implements locks between different fibers in a thread.

home_raft_log_store related function will never run in  fiber, i will run in a separate thread in nuraft.